src/Entity/DatasetSubmission.php

```diff
@@ -954,6 +954,20 @@ class DatasetSubmission extends Entity
      */
     private $datasetLinks;
 
+    /**
+     * For cold-stored, the total unpacked filecount (not dirs).
+     *
+     * @ORM\Column(type="integer", nullable=true)
```

Comment:
The type could also just be text, there may not be a real need to set this to int or bigint unless it a sequence.

Reply:
https://github.com/griidc/pelagos/blob/develop/src/Repository/DatasetRepository.php#L95

Reply:
Then should this type here be bigint?

Reply:
This it not file size but count. But in the case of filesize, I agree we should use bigInt to avoid casting.

Reply:
I can see a future need to get average, max, min filecounts in a report, so I'd rather leave it numeric.